salt/utils/schedule.py

```diff
@@ -659,9 +664,9 @@ def handle_func(self, multiprocessing_enabled, func, data):
             # Reconfigure multiprocessing logging after daemonizing
             log_setup.setup_multiprocessing_logging()
 
-        if multiprocessing_enabled:
-            # Don't *BEFORE* to go into try to don't let it triple execute the finally section.
-            salt.utils.process.daemonize_if(self.opts)
+        #if multiprocessing_enabled:
+        #    # Don't *BEFORE* to go into try to don't let it triple execute the finally section.
+        #    salt.utils.process.daemonize_if(self.opts)
```

Comment:
Do we need this commented-out code around?

Reply:
Nope, good catch. Fixed